spec/components/candidate_interface/application_complete_content_component_spec.rb

```diff
@@ -166,18 +173,20 @@ def stub_application_dates_with_form_editable
     allow(ApplicationDates).to receive(:new).and_return(application_dates)
   end
 
-  def create_application_form_with_course_choices(statuses:)
-    application_form = create(:application_form)
-
-    statuses.each do |status|
-      create(
+  def create_application_form_with_course_choices(statuses:, all_provider_decisions_made: false)
```

Comment:
I don't understand the `all_provider_decisions_made` arg. Isn't it a proxy for the information in the `statuses` arg — ie if the choices are `rejected` and `enrolled` then all provider decisions have been made?

Reply:
@duncanjbrown  You're right, this can also be stubbed as it can be inferred from the statuses passed to the `create_application_form_with_course_choices` method: https://github.com/DFE-Digital/apply-for-postgraduate-teacher-training/pull/1633/commits/c411432d34142c4e5b5ca372af355226e1f06b1b I'll rebase this if you're happy with it.

Reply:
@duncanjbrown https://github.com/DFE-Digital/apply-for-postgraduate-teacher-training/pull/1633/commits/10aa7cb6928fd91167c8605472f29d81376ccaa4 amends the underlying method as discussed.